hooked/views.py

```diff
@@ -49,8 +49,9 @@ def __init__(self, *args, **kwargs):
     def dispatch(self, request, *args, **kwargs):
         return super(WebhookReceiverView, self).dispatch(request, *args, **kwargs)
 
-    def log_failure(self, message, **kwargs):
-        logger.warning(message, extra={'body': self.request.body}, **kwargs)
+    def log_failure(self, message, extra={}, **kwargs):
```

Comment:
Actually the proper thing to do is to use None for any reference type objects (pointers) because even the empty {} can trip you up - as it too is shared. Also I only know about this only because pylint yelled at me for it.